all-rit/ALL
# 546 Fix Lab 4 Broken Exercise

- [x ] No discrepancies across browsers. Eg. chrome vs safari
- [x ] Pages pass W3C Validation for HTML (https://validator.w3.org)
- [ x] CSS (https://jigsaw.w3.org/css-validator/)
- [x ] Google sign-in is operational
- [ x] Accessibility functions
- [x ] Pages can scale without distorting page
- [ x] No dead links
- [ x] Navbar is consistent across the site
- [ x] Pages are screen-reader accessible
- [x ] Contrast meets standards for accessibility
- [x ] Pages are keyboard accessible
- [x ] Passes WAVE Evaluation Tool chrome extension


To test:
- Go to http://localhost:3000/Lab4/Exercise
- Ensure that the "Skip to main content" button actually redirects you to the form
- On this page http://localhost:3000/Lab4/Exercise/FormHintInaccessible, ensure that a "Give Up" button appears after you've tried submitting the form
- On this page http://localhost:3000/Lab4/Exercise/FormHintAccessible, ensure that you can see the Tooltip and that you can tab through it
